JBIDE-26903 - add the Utils class.

zcervink opened the pull request:
Add the Utils class.

please review @jkopriva

Signed-off-by: Zbynek Cervinka <zcervink@redhat.com>

jkopriva commented:
I would also suggest this method as a part o Utils class:
https://github.com/jbosstools/jbosstools-integration-tests/blob/master/tests/org.jboss.tools.examples.ui.bot.test/src/org/jboss/tools/examples/ui/bot/test/integration/AbstractImportQuickstartsTest.java#L311

jkopriva commented on central/test-framework/org.jboss.tools.central.reddeer/src/org/jboss/tools/central/reddeer/utils/Utils.java:
```diff
@@ -0,0 +1,51 @@
+ */
+
+
+public class Utils {
```
I would suggest more meaningful class name like CentralUtils, CentralTestUtils or something similar. Utils is quite general.

zcervink replied:
I have renamed it to CentralUtils.

zcervink commented:
> I would also suggest this method as a part o Utils class:
> https://github.com/jbosstools/jbosstools-integration-tests/blob/master/tests/org.jboss.tools.examples.ui.bot.test/src/org/jboss/tools/examples/ui/bot/test/integration/AbstractImportQuickstartsTest.java#L311

@jkopriva I have moved it too.

jkopriva commented:
Please add javadoc to new functions.

zcervink commented:
> Please add javadoc to new functions.

@jkopriva javadocs added. (The javadoc was not there before I have moved the methods, so I thought they are not needed. Now added).

jkopriva commented:
LGTM
